jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -57,15 +57,16 @@ RUN ${GZDEV_DIR}/gzdev.py repository enable --project=${GZDEV_PROJECT_NAME} --fo
     if [ -n $GZDEV_TRY_BRANCH ]; then git -C ${GZDEV_DIR} checkout $GZDEV_TRY_BRANCH; fi || true && \
     ${GZDEV_DIR}/gzdev.py repository enable --project=${GZDEV_PROJECT_NAME} --force-linux-distro=${DISTRO} )
 DELIM_OSRF_REPO_GZDEV
-else
+fi
+
+# This could duplicate repositories enabled in the step above. gzdev should warn about it without failing.
```

Comment:
I assume this is safe to do. I'm not familiar enough with the code to give it a fair review.

Reply:
It is a bit risky but gzdev should not fail if the repository is duplicated. All the testing jobs using extra repositories or not have been passing just fine. Might be corner cases but should work for most of the builds.